deployer/cluster.py

```diff
@@ -61,6 +61,19 @@ def deploy_support(self, cert_manager_version):
         )
         print_colour("Done!")
 
+        # FIXME: Revert this addition after one upgrade is made
+        print_colour("Preparing for prometheus v16 v19 migration...")
+        subprocess.check_call(
+            [
+                "kubectl",
+                "delete",
+                "daemonset",
+                "--namespace=support",
+                "support-prometheus-node-exporter",
+            ]
+        )
+        print_colour("Done!")
+
```

Comment:
This is the added patch to be removed after merging this with a successful deploy